resources/dataMigration/teams-assignment.json

```diff
@@ -126,16 +126,5 @@
       "mod-remote-storage"
     ],
     "slackChannel": "#volaris-dev"
-  },
```

Comment:
that will brake json structure...

Reply:
![image](https://github.com/folio-org/pipelines-shared-library/assets/151620067/78dc52ea-7121-4495-96c9-74a21b5d19e7)

seems it's valid